web/packages/test/scripts/configure-substrate.sh

```diff
@@ -84,13 +84,20 @@ config_xcm_version() {
     send_governance_transact_from_relaychain $ASSET_HUB_PARAID "$call"
 }
 
+fund_penpal_sender_sovereign() {
+    # forceSetBalance($sender_sovereign_account, 1000000000000)
+    local transact_call="0x0a0800"$sender_sovereign_account"070010a5d4e8"
+    send_governance_transact_from_relaychain $PENPAL_PARAID "$transact_call"
+}
+
```

Comment:
Since the [method is repeated here](https://github.com/Snowfork/snowbridge/pull/1141/files#diff-1c6c806fd0be56e4d5eaa21c28ffcc41567c65dd40c8c470bea4f50556c8efe4R7), maybe we can use the method in `configure-penpal` instead of duplicating.